test/e2e/common.go

```diff
@@ -47,6 +47,30 @@ func createNamespace(ctx context.Context, client testClient, namespace string) e
 	return err
 }
 
+// createNamespaceAndAnnotation creates a kubernetes namespace and an annotation
+func createNamespaceAndAnnotation(ctx context.Context, client testClient, namespace string, annotation string) error {
+	ns := builder.ForNamespace(namespace).Result()
+	_, err := client.clientGo.CoreV1().Namespaces().Create(ctx, ns, metav1.CreateOptions{})
+
+	if apierrors.IsAlreadyExists(err) {
+		return nil
+	}
```

Comment:
At this point, if `err != nil` but it's not an IsAlreadyExists error, then the error should be returned. If the namespace *does* already exist, then we should continue to add the annotation, otherwise the already-existing namespace won't get the annotation added.

Reply:
Style nit: the extra parentheses around `apierrors.IsAlreadyExists(err)` are not needed. Line 55 is more readable as:
```
	if err != nil && !apierrors.IsAlreadyExists(err) {
```